.tractusx

```diff
@@ -1,5 +1,5 @@
 product: "sldt-digital-twin-registry"
```

Comment:
Could you please correct from "sldt-digital-twin-registry" to "Digital Twin Registry"? Then we have the same wording like in our other services.

Reply:
done